src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -187,6 +196,10 @@ public MockResponse dispatch(RecordedRequest request) {
                     return new MockResponse().setBody("[" + filterAttributesAsString + "," + filter2AttributesAsString + "]")
                             .setResponseCode(200)
                             .addHeader("Content-Type", "application/json; charset=utf-8");
+                } else if (path.matches("/v1/elements\\?ids=" + CASE_UUID) && "GET".equals(request.getMethod())) {
+                    return new MockResponse().setBody("[" + caseAttributesAsString + "]")
```

Comment:
Here you mock a response from directory-server.
Here you have specific Metadata in this response.
I don't think it's the directory-server which add those info.